benchmarks/check.mk

```diff
@@ -173,6 +173,3 @@ compressibility_formulations/: dummy
 	@$(def); run_prm $@/lateral_pipe_transient lateral_pipe.prm
 	@$(def); run_prm $@/vertical_pipe vertical_pipe.prm
 
-#https://github.com/geodynamics/aspect/issues/3534
```

Comment:
Do you need to replace this with something that runs the benchmark?

Reply:
No, because the default rule will compile and run all prms.